addon/webextension/background.js

```diff
@@ -94,11 +94,11 @@ class PersistentPageModificationEffect {
     });
   }
 
-  /*
+  /**
    * Returns true only if the URL's protocol is in APPLICABLE_PROTOCOLS.
    */
   protocolIsApplicable(url) {
-    var anchor =  document.createElement('a');
+    var anchor =  document.createElement("a");
```

Comment:
Not sure if we want to force `let` and `const` instead of using `var`... But that's probably a future conversation.